packages/internal/src/config.js

```diff
@@ -2,8 +2,8 @@
 /** @file
  *
  * Some of this config info may make more sense in a particular package. However
- * due to the maxNodeModuleJsDepth hack and our general lax dependency graph,
- * sometimes rational placements cause type resolution errors.
+ * due to #4620 and our lax package dependency graph, sometimes rational
```

Comment:
Please use a full URL instead of issue numbers which break in case of repo migration.